notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>NIT: the dimension of the manifold is n and is equal to the number of degrees of freedom of a point moving on a n-dimensional space.</p><p> </p><p>NIT: a 2-dimensional hypersphere in 3 dimensions describes all the 3D points that lie on the surface of a sphere, and thus have their number of degrees of freedom restricted to 2.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>for the second comment, what do you think of: </p><p> </p><p>For example, a 2-dimensional hypersphere in 3-dimensions describes all the points in 3D space that lie on the surface of a sphere. In other words, a hypersphere describes all of the points that lie in three dimensions that have two degrees of freedom&nbsp;(𝜙,𝜃).</p>